json.json

```diff
@@ -0,0 +1,1551 @@
+[
```

Comment:
What's the purpose of this file?

Reply:
:-S no idea

Reply:
looks like newly added in this PR

Reply:
:-o may be my git aa added it :-S. I'll check!

Reply:
I've deleted the file